ndarray/src/main/java/org/tensorflow/ndarray/impl/sparse/DoubleSparseNdArray.java

```diff
@@ -73,14 +73,16 @@
    *     element in indices. For example, given {@code indices=[[1,3], [2,4]]}, the parameter {@code
    *     values=[18, 3.6]} specifies that element {@code [1,3]} of the sparse NdArray has a value of
    *     {@code 18}, and element {@code [2,4]} of the NdArray has a value of {@code 3.6}.
+   * @param defaultValue Scalar value to set for indices not specified in {@link #getIndices()}
```

Comment:
Maybe this should be `{@code indices}` rather than `{@link #getIndices()}`, as it refers to the construction parameter rather than some instance method that you can't actually call when constructing the object.